ddl/ddl_tiflash_api.go

```diff
@@ -425,7 +425,12 @@ func (d *ddl) PollTiFlashRoutine() {
 				if d.ownerManager.IsOwner() {
 					_, err := d.pollTiFlashReplicaStatus(sctx, pollTiflashContext)
 					if err != nil {
-						logutil.BgLogger().Warn("pollTiFlashReplicaStatus returns error", zap.Error(err))
+						switch err.(type) {
+						case *infosync.MockTiFlashError:
+							// If we have not set up MockTiFlash instance, for those tests without TiFlash, just suppress.
+						default:
+							logutil.BgLogger().Warn("pollTiFlashReplicaStatus returns error", zap.Error(err))
+						}
```

Comment:
Please consider adding a std error in `errno/errcode.go` and using `(e *Error) Equal` to compare the error.

Reply:
This error only happens in test